libs/entry-components/styles/modules/components/tables/_header.scss

```diff
@@ -0,0 +1,9 @@
+@use 'sass:map';
+
+@mixin generate-from($theme) {
+	$header-cell-font-size: map.get($theme, 'tables', 'header', 'font-size');
```

Comment:
Maybe you can put cell-font-size instead of header-cell-font-size, file is called header